src/fem/op_accessors.hh

```diff
@@ -31,8 +31,9 @@ using Tensor = arma::mat33;
 template <class ValueType>
 class ElQ {
 public:
-    /// Forbidden default constructor
-    ElQ() = delete;
+    /// Ddefault constructor
```

Comment:
Comment why default constructor is needed. We can allow it, but must be carefull since that leads to uninitialized instances.